src/control/cmd/daos_server/main.go

```diff
@@ -69,10 +69,7 @@ func exitWithError(log *logging.LeveledLogger, err error) {
 
 func parseOpts(args []string, opts *mainOpts, log *logging.LeveledLogger) error {
 	p := flags.NewParser(opts, flags.HelpFlag|flags.PassDoubleDash)
-	// TODO(DAOS-3129): Remove this when subcommands are required, in order
-	// to eliminate parsing ambiguity.
-	p.Options |= flags.IgnoreUnknown
-	p.SubcommandsOptional = true
+	p.SubcommandsOptional = false
```

Comment:
The zero value of a boolean is false, so just removing this line altogether would have the same effect.

Reply:
yes I was just making it explicitly clear